core/src/test/java/io/neow3j/contract/Hash160Test.java

```diff
@@ -32,18 +33,8 @@ public void createFromValidHash() {
     }
 
     @Test(expected = IllegalArgumentException.class)
-    public void createFromTooShortHash() {
-        new Hash160("23ba2703c53263e8d6e522dc32203339dcd8eee");
-    }
-
-    @Test(expected = IllegalArgumentException.class)
-    public void createFromTooLongHash() {
-        new Hash160("c56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9ba");
-    }
-
-    @Test(expected = IllegalArgumentException.class)
-    public void createFromMidSizedHash() {
-        new Hash160("23ba2703c53263e8d6e522dc32203339dcd8eee9938a3e");
```

Comment:
Why were these tests removed?

Reply:
I also believe they should remain in the test suite.

Reply:
The first two tests don't throw the expected exception because the length is incorrect, but because the hex regex pattern does not match them. Given the test names, I think that is not what is supposed to be tested here. But you're correct, I forgot to replace them accordingly. I'll add more precise tests for that.

For the third test: I don't see the purpose of the `..fromMidSizedHash`. It actually just tests the same as the test with a hash that is too long. Can you tell me, what it was supposed to test, since the naming confused me here?

Reply:
> For the third test: I don't see the purpose of the ..fromMidSizedHash. It actually just tests the same as the test with a hash that is too long. Can you tell me, what it was supposed to test, since the naming confused me here

You're right. You can remove that case. It doesn't test anything else then the `...TooLongHash()` test.